docs/Battery.md

```diff
@@ -102,14 +102,45 @@ If you're using an OSD that expects the multiwii current meter output value, the
 
 ### ADC Sensor
 
-The current meter may need to be configured so the value read at the ADC input matches actual current draw.  Just like you need a voltmeter to correctly calibrate your voltage reading you also need an ammeter to calibrate the current sensor.
+The current meter needs to be configured so the value read at the Analog to Digital Converter (ADC) input matches actual current draw.  Just like you need a voltmeter to correctly calibrate your voltage reading you also need an ammeter to calibrate the current sensor.
+Unlike voltage sensing which is usually quite good from the factory, current sensing varies significantly from board to board and should be calibrated.
 
-Use the following settings to adjust calibration:
+It is recommended to set `multiwii_amperage_meter_output` to `OFF` when calibrating ADC current sensor.
 
-`amperage_meter_scale`
-`amperage_meter_offset`
+To measure the current a linear response device is used that converts the current traveling through it to a voltage to be read by the ADC on your flight controller.
+The maximum voltage that the flight controller can read is 3.3V (3300 mV), this is usually the limiting factor on the maximum measurable current.
+Most sensors use a shunt resistor to measure current however there are a few that use hall effect sensors.
 
-It is recommended to set `multiwii_amperage_meter_output` to `OFF` when calibrating ADC current sensor.
+The flight controller uses the following equation to convert the measured ADC voltage to a current.
+```
+Current(Amps) = ADC (mV) / amperage_meter_scale * 10 + amperage_meter_offset / 1000
+```
+Where the calibrations are:
+
+| Setting                       | Description                                              |
+| ----------------------------- | -------------------------------------------------------- | 
+| `amperage_meter_scale`     | The scaling factor in mV/10A  |
+| `amperage_meter_offset`    | The offset in mA |
+
+This is in the mathematical form of y = x/m + b and with a few measurements along this line you can calibrate any combination of sensor and flight controller to a high accuracy.
+
+#### Calibrate using an ammeter
+
+**!!Important: Always take off your props before doing any testing!!**
+
+To calibrate your flight controller with a current meter follow these steps.
+
+1. Make a copy of [this google sheet](https://docs.google.com/spreadsheets/d/1lkL-X_FT9x2oqrwQEctDsEUhgdY19upNGc78M6FfJXY/). It will do all of the maths for you.
```

Comment:
It's nice to have this in a Google sheet. But I think we should present the formulae that are used as well, for users who can't / don't want to use the sheet.

Reply:
The only important one is already there I think, anyone with the know how to do it on there own should be fine with just that.

The sheet just formats everything nicely and does a linear regression, nothing special hidden in there.

Reply:
Ok. Then we should point that out in the text so it becomes clear to users without having to look at the spreadsheet.